packages/react/jest.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  roots: ['./'],
+  globals: {
+    'ts-jest': {
+      tsConfig: 'tsconfig.test.json',
+    },
+  },
+  modulePathIgnorePatterns: ['<rootDir>/dist/'],
+  moduleNameMapper: {
+    '@clr/(.*)': '<rootDir>/../../dist/$1',
```

Comment:
This is so that test files can resolve all the `import { x } from "@clr/core/y"` type of statement. I could make it more specific to `@clr/core` in the regex if needed. For now this seems sufficient.